src/control/controlbuttonmode.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+enum class ControlButtonMode {
+    PUSH = 0,
+    TOGGLE,
+    POWERWINDOW,
+    LONGPRESSLATCHING,
+    TRIGGER
+};
```

Comment:
since we're already touching every spot anyways. lets adhere to the core guidelines:
[Enum.5: Don’t use ALL_CAPS for enumerators](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps) and [Enum.8: Specify enumerator values only when necessary](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-value)
```suggestion
enum class ControlButtonMode {
    Push,
    Toggle,
    PowerWindow,
    LongPressLatching,
    Trigger
};
```